distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/annotations/PaxExamRule.java

```diff
@@ -62,6 +64,12 @@ public PaxExamRule(Object testInstance) {
         this.testInstance = testInstance;
     }
 
+    public static void failWithThrowableStacktrace(Throwable t, String message) {
```

Comment:
I botched it a bit in my last refactoring, but the intent was to avoid coupling in the other direction, since `PaxExamRule` also should use this helper method, so I was trying to avoid a circular dependency. I can create a separate class just for this method if you'd prefer.